chart/README.md

```diff
@@ -25,3 +25,4 @@ metrics, observability, health checks, with pre- and post-deployment evaluations
 | `global.commonLabels`           | Common labels to add to all Keptn resources. Evaluated as a template      | `{}`      |
 | `global.commonAnnotations`      | Common annotations to add to all Keptn resources. Evaluated as a template | `{}`      |
 | `global.caInjectionAnnotations` | CA injection annotations for cert-manager.io configuration                | `{}`      |
+| `global.isOpenshift`            | Handles specific settings for Openshift                                   | `false`   |
```

Comment:
are you expecting that more openshift-specific settings will be needed, since the docs comment here seems to suggest that?
then i would go straight to `global.openShift.enabled` so that we have an easy option to add more values below `global.openShift`. Otherwise I'd still like to rename the value to `global.openShiftEnabled` here and in the other charts

Reply:
I would vote for `global.openShift.enabled` here to have an easy option to extend the setting in the future

Reply:
Looking at what we've done in other projects (like cert-manager) we usually end up with a single switch, but I really don't know how this will work out in keptn so I like your idea of global.openshift.enabled. I'll update.